### PR TITLE
Feature/studybox 11 pobranie talii do pamieci

### PR DESCRIPTION
**Pull request by Bartosz-Kozajda**

### CODE REVIEW

File: app/src/main/java/com/blstream/studybox/api/DecksApplication.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * Created by Bartosz Kozajda on 08.03.2016.
+ */
+public class DecksApplication extends Application {
+    private DecksManager decksManager;
+    private Bus bus = BusProvider.getInstance();
+
+    @Override
+    public void onCreate() {
+        decksManager = new DecksManager(this, bus);
```
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/api/DecksApplication.java]: You don't forget about calling super.onCreate() ?

File: app/src/main/java/com/blstream/studybox/api/DecksManager.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import retrofit.Callback;
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+
+/**
+ * Created by Bartosz Kozajda on 08.03.2016.
+ */
+public class DecksManager {
+    private Context context;
```
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/api/DecksManager.java]: Now it's useless, let me now if we need this in future.

File: app/src/main/java/com/blstream/studybox/api/RestClient.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.blstream.studybox.model.DecksList;
+import retrofit.Callback;
+import retrofit.RestAdapter;
+
+/**
+ * Created by Bartosz Kozajda on 08.03.2016.
+ */
+public class RestClient {
+    private static final String BASE_URL = "http://private-5f2e4b-studybox2.apiary-mock.com/";
```
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/api/RestClient.java]: Maybe move this to holder like Constants.

File: app/src/main/java/com/blstream/studybox/api/RestClient.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    private static final String API_KEY = "json";
+
+    private static RestClient restClient;
+    private static RestAdapter restAdapter;
+
+    public static RestClient getClient(){
+        if(restClient == null)
```
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/api/RestClient.java]: I suggest you to use curly bracers even if we have one line of code in statement.

**Comment by Bartosz-Kozajda** [comment]
Can you make review again? @maras93

File: app/src/main/java/com/blstream/studybox/api/RestClientManager.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package com.blstream.studybox.api;
+
+import android.content.Context;
```
Comment (maras93) [inline app/src/main/java/com/blstream/studybox/api/RestClientManager.java]: Remove unused import

**Pull request merged**